app/src/tests/components/contents/PacketGroup/ToggleParamColumnsDropdown.test.tsx

```diff
@@ -0,0 +1,59 @@
+import { fireEvent, render, screen, waitFor } from "@testing-library/react";
+// eslint-disable-next-line max-len
+import { ToggleParamColumnsDropdownColumns } from "../../../../app/components/contents/PacketGroup/ToggleParamColumnsDropdown";
```

Comment:
Would this line still be too long if you did it like 
```suggestion
import {
  ToggleParamColumnsDropdownColumns
} from "../../../../app/components/contents/PacketGroup/ToggleParamColumnsDropdown";
```
seems a shame to have to put the lint exception for this sort of stuff!
Could we have an `@` type alias for source root?

Reply:
its an issue with prettier it prefers single imports on 1 line https://github.com/prettier/prettier/issues/1954...

i tried doing alias ages before when did rewrite, but didn't work properly because it uses old version of react... think we have to upgrade project to vite!! made a ticket https://mrc-ide.myjetbrains.com/youtrack/issue/mrc-6193/Upgrade-FE-to-vite